website/www/site/content/en/documentation/io/connectors.md

```diff
@@ -252,7 +252,10 @@ This table provides a consolidated, at-a-glance overview of the available built-
       ✔
       <a href="https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/kinesis/KinesisIO.html">native</a>
     </td>
-    <td>Not available</td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/pydoc/current/apache_beam.io.kinesis.html#apache_beam.io.kinesis.ReadDataFromKinesis">native</a>
```

Comment:
This is actually exposed via cross-language: https://github.com/apache/beam/blob/ddfb6907288369bfc71c3a310cb9ceb717a4d3a0/sdks/python/apache_beam/io/kinesis.py#L196-L201

(note the "ExternalTransform")
```suggestion
      <a href="https://beam.apache.org/releases/pydoc/current/apache_beam.io.kinesis.html#apache_beam.io.kinesis.ReadDataFromKinesis">via X-language</a>
```

Reply:
Fixed